authLdap.php

```diff
@@ -263,39 +262,40 @@ function authLdap_login($user, $username, $password, $already_md5 = false)
         }
 
         $uid = authLdap_get_uid($realuid);
-        $role = '';
-
-        // we only need this if either LDAP groups are disabled or
-        // if the WordPress role of the user overrides LDAP groups
-        if (!$authLDAPGroupEnable || !$authLDAPGroupOverUser) {
-            $role = authLdap_user_role($uid);
-        }
+        $roles = authLdap_user_roles($uid);
```

Comment:
We definitely need this if-condition. Otherwise the LDAP-roles will ALWAYS overwrite the Wordpress-roles even though people might not want this plugin to handle the roles.